pull-request.py

```diff
@@ -41,6 +41,11 @@ def parse_into_list(values):
     return [x.strip() for x in values.split(" ")]
 
 
+def set_env(name, value):
+    """helper function to echo a key/value pair to the environement file"""
+    os.system('echo "%s=%s" >> $GITHUB_ENV' % (name, value))
+
+
```

Comment:
What about using an implementation like this instead?
```python
def set_env(name, value):
    environment_file_path = os.environ.get('GITHUB_ENV')

    with open(environment_file_path, "a") as environment_file:
        environment_file.write("%s=%s" % (name, value))
```

Reply:
Yes this is a much better idea, I'll update the PR with this for you to test!